data-prepper-plugins/avro-codecs/src/test/java/org/opensearch/dataprepper/plugins/codec/avro/AvroOutputCodecTest.java

```diff
@@ -75,7 +75,6 @@ void constructor_throws_if_schema_is_invalid() {
         RuntimeException actualException = assertThrows(RuntimeException.class, this::createObjectUnderTest);
 
         assertThat(actualException.getMessage(), notNullValue());
-        assertThat(actualException.getMessage(), containsString(invalidSchema));
```

Comment:
Why did we remove this assertion?

Reply:
I gave a very brief explanation in the description. Prior to Jackson 2.16, Jackson would include input data in the messages provided. Now it does not do that (by default) to avoid possible log injection or display of sensitive data. So this exception fails now as the input schema is no longer in the message.